docs/2.7.0/docs/app-dev/authorization.rst

```diff
@@ -73,58 +73,67 @@ The Daml Ledger API uses the following rights to govern request authorization:
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| CommandCompletionService            | CompletionEnd                 | public                                                 |
+|                                     +-------------------------------+--------------------------------------------------------+
+|                                     | CompletionStream              | for each requested party p: canReadAs(p)               |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| CommandSubmissionService            | Submit                        | for submitting party p: canActAs(p)                    |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| CommandService                      | All                           | for submitting party p: canActAs(p)                    |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| Health                              | All                           | no access token required for health checking           |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| IdentityProviderConfigService       | All                           | participant_admin                                      |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| LedgerConfigurationService          | GetLedgerConfiguration        | public                                                 |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| MeteringReportService               | All                           | participant_admin                                      |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| PackageService                      | All                           | public                                                 |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| PackageManagementService            | All                           | participant_admin                                      |
++-------------------------------------+-------------------------------+--------------------------------------------------------+
+| PartyManagementService              | All                           | participant_admin                                      |
+|                                     +-------------------------------+--------------------------------------------------------+
```

Comment:
Hard to know for sure, but it seems to me that we need here
```suggestion
+-------------------------------------+-------------------------------+--------------------------------------------------------+
```

Reply:
Yes, looks better with those extra dividers